docs/documentation/maya/simshape.md

```diff
@@ -165,6 +198,7 @@ Simshape can emulate the behaviour of facial muscles by computing the muscle act
         Activations are not computed. This option is selected by default.
```

Comment:
Should be this line here?

Reply:
I am not sure I follow here, why shouldn't this line be here?